l10n/bundle.l10n.zh-CN.json

```diff
@@ -52,149 +52,149 @@
-  "Create ESP-IDF From Extension Templates": "从扩展模板创建 ESP-IDF",
+  "Error encountered while adding dependency {dependency} to the component \"{component}\"": "将依赖项 {dependency} 添加到组件 \"{component}\" 时出错",
+  "Error encountered while creating project from example \"{example}\"": "基于 \"{example}\" 示例创建项目时出错",
+  "Create ESP-IDF From Extension Templates": "基于扩展模板创建 ESP-IDF",
   "Install ESP-ADF": "安装 ESP-ADF",
   "Install ESP-MDF": "安装 ESP-MDF",
   "Install ESP-Matter": "安装 ESP-Matter",
   "Install ESP-Rainmaker": "安装 ESP-Rainmaker",
   "Project Configuration editor": "项目配置编辑器",
   "Select Project Configuration": "选择项目配置",
-  "Install Extension Python Requirements": "安装扩展 Python 要求",
-  "Install ESP-Matter Python Requirements": "安装 ESP-Matter Python 要求",
-  "Add .vscode subdirectory files": "添加.vscode子目录文件",
-  "Add .devcontainer subdirectory files": "添加.devcontainer子目录文件",
+  "Install Extension Python Requirements": "为扩展安装 Python 依赖包",
+  "Install ESP-Matter Python Requirements": "安装 ESP-Matter Python 依赖包",
+  "Add .vscode subdirectory files": "添加 .vscode 子目录文件",
+  "Add .devcontainer subdirectory files": "添加 .devcontainer 子目录文件",
   "Create ESP-IDF Component": "创建 ESP-IDF 组件",
-  "Flash with JTAG": "带 JTAG 的闪存",
-  "Flash with UART": "带UART的闪存",
-  "Flash with DFU": "带 DFU 的闪存",
-  "Launch Websocket server and IDF Monitor": "启动 Websocket 服务器和 IDF 监视器",
+  "Flash with JTAG": "通过 JTAG 接口烧录项目",
+  "Flash with UART": "通过 UART 接口烧录项目",
+  "Flash with DFU": "通过 DFU 接口烧录项目",
+  "Launch Websocket server and IDF Monitor": "启动 WebSocket 服务器和 IDF 监视器",
   "Start/Stop QEMU Server": "启动/停止 QEMU 服务器",
   "Configure ESP-IDF Extension": "配置 ESP-IDF 扩展",
-  "Select current ESP-IDF version": "选择当前 ESP-IDF 版本",
-  "Show Examples": "显示示例",
-  "New Project Wizard": "新建项目向导",
-  "Select Current Project workspace folder": "选择当前项目工作区文件夹",
-  "Select Port to Use (COM, tty, usbserial)": "选择要使用的端口（COM、tty、usbserial）",
-  "Set Espressif Device Target (IDF_TARGET": "设置 Espressif 设备目标 (IDF_TARGET)",
-  "SDK Configuration Editor (menuconfig)": "SDK 配置编辑器（menuconfig）",
+  "Select current ESP-IDF version": "选择当前的 ESP-IDF 版本",
+  "Show Examples": "展示示例项目",
+  "New Project Wizard": "新项目向导",
+  "Select Current Project workspace folder": "选择当前项目的工作区文件夹",
+  "Select Port to Use (COM, tty, usbserial)": "选择要使用的端口 (COM、tty、usbserial)",
+  "Set Espressif Device Target (IDF_TARGET": "设置乐鑫设备目标 (IDF_TARGET)",
+  "SDK Configuration Editor (menuconfig)": "SDK 配置编辑器 (menuconfig)",
   "Build Project": "构建项目",
-  "ESP-IDF Size": "ESP-IDF 尺寸",
-  "Full Clean": "清除构建",
-  "ESP-IDF: Select Flash Method": "ESP-IDF：选择烧录方法",
-  "Flash Device": "烧录设备",
-  "Erase Flash": "擦除闪存",
-  "Monitor Device": "监控设备",
-  "ESP-IDF: Build, Flash and Monitor": "ESP-IDF：构建、烧录和监控",
+  "ESP-IDF Size": "ESP-IDF 二进制文件大小分析",
+  "Full Clean": "彻底清理",
+  "ESP-IDF: Select Flash Method": "ESP-IDF：选择烧录方式",
+  "Flash Device": "烧录项目",
```

Comment:
Hi @April-Yjj Actually, here "Flash Device" emphasizes the process of flashing a built project to a selected device, rather than referring to the device itself. I expect this command to appear on the bottom blue menu bar when you hover your mouse over the following sign (the one in the middle):
![image](https://github.com/user-attachments/assets/b17bc99c-c8ba-4ad3-b1eb-bfc90b8ce776)

Reply:
Hi @zztiger123 Could you help confirm the meaning of this command? Does it mean ``choosing the device to flash``（选择烧录设备）? or the flash action（烧录项目）? Thank you!